src/tools/kata-ctl/src/check.rs

```diff
@@ -100,8 +101,26 @@ pub fn check_cpu_attribs(
     Ok(missing_attribs)
 }
 
-pub fn run_network_checks() -> Result<()> {
-    Ok(())
+#[tokio::main]
+pub async fn run_network_checks() -> Result<()> {
+    println!("Running network checks...");
+    if let Ok((connection, handle, _)) =
+            rtnetlink::new_connection().context("failed to create netlink connection")
+        {
+            let thread_handler = tokio::spawn(connection);
+            defer!({
+                thread_handler.abort();
+            });
+
+            handle
+                .link()
+                .add()
+                .veth("foo".to_string(), "bar".to_string());
+        }
+    else {
+        return Err(anyhow!("cannot create veth pair"));
+    }
+    return Ok(())
```

Comment:
Use expression return value `Ok(())` instead a declarative return value `return Ok(())`

Reply:
Yep - the last expression in a function is the return value so you don't need the `return` here.

I'd recommend setting up `rust-analyzer` with your editor as it will tell you about these sorts of "nits".

Reply:
Thank you. I also fixed this issue. @Ji-Xinyou, @jodh-intel